README.md

```diff
@@ -2,7 +2,64 @@
 
 ## About
 
-This repository defines how to store geospatial information (point, lines, polygons) in Parquet.
+This repository defines how to store geospatial information (point, lines, polygons) in [Parquet](https://parquet.apache.org/), a popular 
+[columnar storage format](https://en.wikipedia.org/wiki/Column-oriented_DBMS). Our goal is to standardize how geospatial data is represented
+in Parquet to further geospatial interoperability among tools using Parquet today, and hopefully help push forward what's possible with 
+'cloud-native geospatial' workflows. 
 
 **Warning:** This is a completely unreleased specification, with several outstanding updates to make. If you are excited about the potential 
-please sound in on the issues and contribute PR's!
+please sound in on the issues and contribute PR's! But please do not start converting all your data to this format, as it will change 
+(and possibly in a backwards incompatible way, see the [versioning](#versioning) section below.
+
+Early contributors include developers from Voltron Data, Microsoft, Carto, Azavea, Planet & Unfolded. Anyone is welcome to join us, by building 
+implementations, trying it out, giving feedback through issues and contributing to the spec via pull requests.
+
+Note that verison 0.1 only supports 2d geometries, as we are not supporting [EWKB](https://libgeos.org/specifications/wkb/#extended-wkb) or 
+[ISO WKB](https://libgeos.org/specifications/wkb/#iso-wkb). We expect to expand WKB to support 3D in the future, if you have thoughts please 
+sound in on #18 and/or follow along there.
+
+## Goals
+
+There are a few core goals driving the initial development.
+
+* **Enable interoperability among cloud data warehouses** - BigQuery, Snowflake, Redshift and others all support spatial operations but importing and exporting data 
+ with existing formats can be problematic. All support and often recommend Parquet, so defining a solid GeoParquet can help enable interoperability.
```

Comment:
Or, put differently, we are maybe missing a goal here. One of the main reasons we started using this in GeoPandas, is to have fast/efficient, columnar file format to store geospatial vector data (in addition to the traditional shapefile / geopackage / geojson). Or maybe that doesn't need to be listed explicitly? (something like "Enable the Parquet file format to store geospatial data" is already covered by the very first paragraph of this README?)

Reply:
Ok, made an attempt at this. Ended up splitting out 'goals' from 'features', and made a stab at explaining it. I've not been deep in the new columnar workflows, so any tweaks to better explain the potential are more than welcome.

Reply:
> Build a fast/efficient, columnar file format to store geospatial vector data to enable XXXX
>
> I can use help on what it enables...

I think in the first place, it enables "fast/efficient _data access_", I would say 
(and then also other features from Parquet, like very good compression (so small file sizes), cheap reading of a subset of columns (the columnar nature), the type system (eg nested types), filtering chunks based on column statistics, ..)

But taking a look at your updates now!

Reply:
> (and then also other features from Parquet, like very good compression (so small file sizes), cheap reading of a subset of columns (the columnar nature), the type system (eg nested types), filtering chunks based on column statistics, ..)

I like these, may try to call them out in 'features', as that mostly focused on the geospatial stuff, but this stuff is good to know.

Reply:
Ok, just committed an attempt to represent some of that. I don't understand the advantages of nested types, so didn't try to add that in. Check to see if I got anything wrong, but it was nice to call out some non-geo 'features'. I also added a bit about it not being so good at write-heavy workflows. Let me know if that's off - I did appreciate how flatgeobuf makes it clear what you should not be using it for...

Reply:
> I don't understand the advantages of nested types, so didn't try to add that in

It's probably not the most important to mention here. In general, Parquet has a wide support for different types, including nested types. So you can for example have "structs" or "lists" in the column values. 
For example it also gets mentioned here: https://databricks.com/glossary/what-is-parquet ("Supports complex data types and advanced nested data structures.")

Reply:
Ah, that's cool. Some geospatial people love really complex data structures, GML went nuts with that stuff. I think we should definitely focus on just the simple features use case and do that really well, but good to know that more complex data structures can be supported.

Reply:
https://databricks.com/glossary/what-is-parquet is a great overview - I think we should include a link to it somewhere in the readme. It includes a decent overview of columnar formats. I'll add it in.

Reply:
I think the nested types are mostly relevant when talking about an arrow-native geometry encoding. For example a nested list of values would be able to describe [a column of `Point`s](https://github.com/geopandas/geo-arrow-spec/pull/12/files#diff-6a6ea8d80b9c2fea0c9877dc99e6670c16a787efdf4729056cb0bc5f3ad4e48fR61-R66), while still having a favorable in-memory encoding (i.e. flat arrays underneath).

Reply:
Very cool. So we'll leave them off here for now, but maybe mention in the future.

Reply:
We can indeed use those nested data types to store geometries (the arrow-native encoding proposal), but just a note that there are ecosystems where they make heavy use of the nested schemas that Parquet enables (eg I think Spark supports this quite well; I am not super familiar with it, since nested columns are not really supported in python/pandas). For example, it can map nicely to structured data you might encounter in json files, logs, etc.

(this is just a clarification, agreed it is not that important to have it included in this PR)